src/SDK/Client/Platform/Platform.ts

```diff
@@ -132,12 +133,17 @@ export class Platform {
 
         this.client = options.client;
 
-        const dppForParsingContracts = new DashPlatformProtocol();
-        const stateRepository = new StateRepository(this.client, dppForParsingContracts);
+        this.dppForParsingContracts = new DashPlatformProtocol();
+        const stateRepository = new StateRepository(this.client, this.dppForParsingContracts);
 
         this.dpp = new DashPlatformProtocol({
             stateRepository,
             ...options,
         });
     }
+
+    async initialize() {
```

Comment:
Let's call initialize in Client methods - they are all async so we can init DPP in lazy way.